### PR TITLE
add nodePort option for extraPorts

### PR DESCRIPTION
**Pull request by johnynfulleffect**
I noticed that unlike other instances in the charts to expose services, you cannot specify a specific `nodePort`, if required, on the `extraPorts` service collection.  I had to edit the svc in kubernetes to specify this directly.  This PR will allow you to specify that in the `values.yaml` and avoid that direct edit in the kubernetes manifests to be overridden.

### CODE REVIEW

File: charts/minecraft-proxy/templates/extraports-svc.yaml [hunk 1]
```diff
@@ -35,6 +35,8 @@ spec:
   - name: {{ .name }}
     port: {{ .service.port }}
     targetPort: {{ .name }}
+    {{- if .service.nodePort }}
```
Comment (itzg) [inline charts/minecraft-proxy/templates/extraports-svc.yaml:38]: It looks like this `if` is missing an `end`

There's three more like this.

**Review by itzg — CHANGES_REQUESTED** [review]

File: charts/minecraft-proxy/templates/extraports-svc.yaml (hunk 1 above)
Comment (johnynfulleffect) [reply charts/minecraft-proxy/templates/extraports-svc.yaml:38]: Thanks!  Totally new to if blocks in helm templates.  Fixed!

File: charts/minecraft/templates/minecraft-svc.yaml [hunk 2]
```diff
@@ -40,6 +40,8 @@ spec:
   - name: {{ .name }}
     port: {{ .service.port }}
     targetPort: {{ .name }}
+    {{- if .service.nodePort }}
```
Comment (itzg) [inline charts/minecraft/templates/minecraft-svc.yaml:43]: This one is also missing an `end`

**Comment by itzg** [comment]
Can you also bump the minor version on the two charts? That's what the verification is failing on.

File: charts/minecraft/templates/minecraft-svc.yaml (hunk 2 above)
Comment (itzg) [inline charts/minecraft/templates/minecraft-svc.yaml:43]: It still looks like this is missing an `{{- end}}`

**Review by itzg — CHANGES_REQUESTED** [review]

File: charts/minecraft/templates/minecraft-svc.yaml (hunk 2 above)
Comment (johnynfulleffect) [reply charts/minecraft/templates/minecraft-svc.yaml:43]: Thanks!  Don't know how I missed that one.

**Comment by itzg** [comment]
Oops, there's now merge conflicts, I'm guessing because the chart version got bumped by other PRs.

**Comment by johnynfulleffect** [comment]
> Oops, there's now merge conflicts, I'm guessing because the chart version got bumped by other PRs.

Thanks!  Resolved conflicts

**Review by itzg — APPROVED** [review]

**Pull request merged**